wbooks/src/app/components/GenericText/styles.ts

```diff
@@ -0,0 +1,55 @@
+import { AppColors } from '@constants/colors';
+import { StyleSheet } from 'react-native';
```

Comment:
for convention we first define all libraries imports and then we call our alias' imports. Reorder this imports